data/multiqc/multiqc.debian.yaml

```diff
@@ -18,6 +18,7 @@ identifiers:
   debian: multiqc
   doi:
   - 10.1093/bioinformatics/btw354
+  - https
```

Comment:
Hmm, this happen because of this https://github.com/bio-tools/content/blob/master/data/multiqc/bioconda_multiqc.yaml#L7


`- doi:https://doi.org/10.1093/bioinformatics/btw354` is not a correct syntax, it contains 2 separators ```:```